### DISCUSSION TITLE
How can I reset the default CSS in quasar?

### DISCUSSION THREAD

**Discussion by YongDevv** [quasarframework/quasar#17343] [General - Components / Directives / etc]
I would like to create a new CSS guide by maintaining the functions supported by quasar and initializing all CSS. Is it possible to unlink only the default CSS that is automatically linked?
![asdasds](https://github.com/quasarframework/quasar/assets/96756001/2a21c4a9-0472-4d8c-b933-3d7770b788bf)

**Comment by ivanjaros** [comment]
Probably only via tweaking the config and excluding the compiled app css.

**Reply by YongDevv** [reply]
I don't know how to exclude CSS from being automatically compiled. Commenting out all the CSS in quasar.config.js didn't help.
I would like to find a way to unlink it rather than overwriting it with a new css file and initializing it.

**Reply by ivan006** [reply]
legends have it there is a way... frodo of the shire. Simply dont import vendor.css into your final build.

**Reply by TobyMosque** [reply]
create a alias to the quasar css who points to a empty file.
```js
build: {
  alias: {
    'quasar/dist/quasar.sass': path.join(__dirname, './path/to/your/styles.sass')
  }
}
```

**Reply by ivan006** [reply]
oh yes i also concluded that but forgot to say. thanks!
